.github/workflows/update_version.yml

```diff
@@ -20,54 +20,36 @@ jobs:
       - name: Get current date for the first release candidate
         if: github.event_name == 'schedule'
         run: |
-          echo "VERSION=$(date +'v%y.%-m')rc0" >> $GITHUB_ENV
-          echo "NON_RC_VERSION=$(date +'v%y.%-m')" >> $GITHUB_ENV
+          echo "VERSION=$(date +'v%y.%-m' + ".0")" >> $GITHUB_ENV
 
       - name: Get current date for a manual release
         if: github.event_name == 'workflow_dispatch'
         run: |
           echo "VERSION=$(date +'v%y.%-m')${{ github.event.inputs.append_to_tag }}" >> $GITHUB_ENV
```

Comment:
This should be be updated to account for the new ".X" format in version.

Reply:
This was because I was assuming that `append_to_tag` was `.1`, `.2`, etc.

As discussed above, I will remove the "workflow_dispatch" portion and simplify this process further